### PR TITLE
LINK-1888 | 3/4 Create price group page

### PR DESCRIPTION
**Pull request by jorilindell**
## Description
Page to create new price groups. Creating price group need financial admin permissions to the publisher organization of the new price group

This is 3rd of the PRs to implement price group admin pages. All PRs are:
- https://github.com/City-of-Helsinki/linkedcomponents-ui/pull/344
- https://github.com/City-of-Helsinki/linkedcomponents-ui/pull/345
- https://github.com/City-of-Helsinki/linkedcomponents-ui/pull/346
- https://github.com/City-of-Helsinki/linkedcomponents-ui/pull/347

## Partially closes
[LINK-1888](https://helsinkisolutionoffice.atlassian.net/browse/LINK-1888)

## Screenshot
Screenshot 2024-03-03 at 22 10 20

[LINK-1888]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-1888?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr346-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedcomponents-pr346-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by harriris-vincit** [comment]
Got a JS error and an empty page when I first created a new price group and then deleted it using the table action.

**Review by harriris-vincit — CHANGES_REQUESTED** [review]
Got a JS error and an empty page when I first created a new price group and then deleted it using the table action. Need to fix the bug.

**Comment by jorilindell** [comment]
> Got a JS error and an empty page when I first created a new price group and then deleted it using the table action. Need to fix the bug.

I'm assuming that the reason for this bug is that the only price group from the last page is deleted and after that API return 404 error and response meta data is not found. I fixed this for price groups by redirecting user to page 1 after delete. This same issue should be fixed to other tables as well. But I will create a separate ticket for that.

File: src/domain/places/placeList/PlaceList.tsx [hunk 1]
```diff
@@ -100,7 +100,7 @@ const PlaceListContainer: React.FC = () => {
   const { count, onSearchSubmit, ...listProps } = useCommonListProps({
     defaultSort: DEFAULT_PLACE_SORT,
     listId: placeListId,
-    meta: placesData?.places.meta,
+    meta: placesData?.places?.meta,
```
Comment (harriris-vincit) [inline src/domain/places/placeList/PlaceList.tsx:103]: This seems a bit unrelated to the price group changes so perhaps could be in its own commit?

**Review by harriris-vincit — APPROVED** [review]
Suggested adding some changes to their own commit, but otherwise looks good to me.

**Pull request merged**